Multisig/UI/App/ViewControllerFactory.swift

```diff
@@ -17,19 +18,29 @@ enum ViewControllerFactory {
     static func rootViewController() -> UIViewController {
         let tabBarVC = MainTabBarViewController()
 
-        if let updateViewController = App.shared.updateController.makeUpdateAppViewController() {
```

Comment:
@MouazAlzahabi this was disabled in release branch, and I did not return it back because this is a buggy implementation as it can allow to start using app without accepting terms